packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor.cs

```diff
@@ -62,24 +62,26 @@ public void UpdateTableData(int numberOfRows)
             var tableData = new Person[numberOfRows + 1];
             for (int i = 0; i < numberOfRows; i++)
             {
-                tableData[i] = new Person(Faker.Name.First(), Faker.Name.Last());
+                tableData[i] = new Person(Faker.Name.First(), Faker.Name.Last(), "https://nimble.ni.dev");
```

Comment:
Is there a reason why you chose two different approaches between the Blazor & Angular example apps? Specifically, in the Angular example app you added a two new fields to the data model (one for the label and one for the URL), and in the Blazor example app you only added one for the URL and are reusing an existing string for the label.

I ask because we have #978 to make the two example apps more consistent when it comes to the table, and this seems to be going in the opposite direction. Is there a technical reason why they need to be different that we should work on addressing?

Reply:
No reason. I've made the Blazor one match the Angular one. The Angular example still differs in that it uses a different url to exercise the router integration, but I think that's reasonable.